src/client/resources/assets/snapper/lang/pt_br.json

```diff
@@ -0,0 +1,42 @@
+{
+  "key.snapper.panorama": "Faça Capturas de Tela panorâmicas",
```

Comment:
Since they're keybinds, they should say "Fazer" instead of "Faça", think of "Do [action]" vs. "Be able to do [action]"